docs/managed-tests/security.md

```diff
@@ -116,3 +116,28 @@ Use suppression judiciously. Strive to follow recommended practices rather than
 
 - **Combine with other tests:**  
   Security tests complement other managed tests, end-to-end tests, and code quality checks, ensuring a holistic view of your extension’s health.
+
+## What do the audit results mean?
+
+Audit results like:
+
+- Known vulnerability check passed
+- Dependent packages vulnerability check passed
+
+Means that the product has successfully passed automated security checks that scan for known risks in its code and any third-party tools it uses.
+
+**What’s being checked?**
+
+We use trusted auditing tools like `composer audit` and `npm audit` to:
+
+- Check the product’s own code for any known security issues.
+- Scan third-party packages and libraries the product depends on — for example, payment gateways, form handlers, or UI components — and ensure none are listed in public vulnerability databases (such as GitHub Security Advisories or the National Vulnerability Database).
```

Comment:
Would it be good to add links to the [GitHub Security Advisories](https://github.com/advisories) and [National Vulnerability Database](https://nvd.nist.gov/) pages?